DenisZackharov/teacherss
# [#1] add docker

### Summary

Add docker and docker compose cofigure

### How it works

None.

### Test plan

List of steps to manually test introduced functionality:

1. Run `bin/docker-setup`
2. Run `bin/docker-server`
4. Check in `localhost:3000`

### Review notes

While reviewing pull-request (especially when it's your pull-request),
please make sure that:

- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- DB schema changes reflect new migrations
- newly introduced DB fields have indexes and constraints
- there are no missed files (migrations, view templates)
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes

### Deploy notes

None.
